public/assets/js/styleguide.js

```diff
@@ -72,4 +72,12 @@ document.addEventListener('DOMContentLoaded', function() {
       showDoc.checked = false;
   });
 
+  var reset = document.getElementById('reset');
+  reset.addEventListener('click', function() {
+    document.getElementById('documentation-button').checked = false;
+    document.getElementById('challenge-button').checked = false;
+    editor.setValue("");
+    ajax();
```

Comment:
I guess this is for clear the iframe generated from whatever code that was in the editor. What do you think of searching for the iframe and setting its src attribute to an empty string?, in that way we wouldn't need to to do an AJAX request.